### PR TITLE
Of/policyfiles

### PR DESCRIPTION
**Pull request by oferrigni**
Will repin before merge.

### CODE REVIEW

**Comment by oferrigni** [comment]
Will update pinnings when the following have landed.

 https://github.com/chef/enterprise-chef-server-schema/pull/22
 https://github.com/chef/oc_erchef/pull/78
 https://github.com/chef/oc-chef-pedant/pull/89
 https://github.com/chef/chef-pedant/pull/96

**Comment by tylercloke** [comment]
LGTM once updated with proper branches.

File: files/private-chef-cookbooks/private-chef/attributes/default.rb [hunk 1]
```diff
@@ -280,7 +280,7 @@
 default['private_chef']['lb']['xdl_defaults']['couchdb_association_requests'] = false
 default['private_chef']['lb']['xdl_defaults']['couchdb_organizations'] = false
 default['private_chef']['lb']['xdl_defaults']['couchdb_associations'] = false
-default['private_chef']['lb']['xdl_defaults']['policies'] = false
+default['private_chef']['lb']['xdl_defaults']['policies'] = true
```
Comment (sdelano) [inline files/private-chef-cookbooks/private-chef/attributes/default.rb]: Is this on by default? Is this just here for testing?

**Comment by tylercloke** [comment]
:+1: pending release notes and changelog update

**Pull request merged**